fspiop/v1_0/openapi3/components/schemas/Amount.yaml

```diff
@@ -8,3 +8,4 @@ description: >-
   minor currency unit. It also only allows four digits in the minor currency
   unit; a negative value is not allowed. Using more than 18 digits in the major
   currency unit is not allowed.
+example: 123.45
```

Comment:
there is little inconsistency over the examples: sometimes string field example value is in quotes, sometimes not. I suggest using the quotes for all strings to distinguish values from numbers

Reply:
thanks for pointing this out. done